contracts/src/v0.8/keystone/test/CapabilityRegistry_DeprecateCapability.t.sol

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: MIT
```

Comment:
nit: file name correct?

Reply:
This tests `deprecateCapability`, so I'm following the convention of naming files with the function that is being tested.